cli/tools/jupyter/server.rs

```diff
@@ -490,9 +490,9 @@ async fn get_jupyter_display(
     )
     .await?;
 
-  if let Some(exception_details) = &response.exception_details {
-    // TODO(rgbkrk): Return an error in userspace instead of Jupyter logs
-    eprintln!("Exception encountered: {}", exception_details.text);
+  if response.exception_details.is_some() {
```

Comment:
I think what we should do instead is change the function above to something like so. This should allow us to determine the difference between an object with no display function and an error.

```
function (object) {
  const display = object[Symbol.for("Jupyter.display")]
  if (display) {
    return JSON.stringify(display());
  } else {
    return null;
  }
}
```

Reply:
Ok cool. I'm still getting a feel for how much we inject in js vs rust.